script/dist.js

```diff
@@ -69,6 +69,7 @@ remove(outDir)
         const meta = {bundles}
         return writeFile(join(outDir, 'meta.json'), JSON.stringify(meta, null, 2), encoding)
       })
+      .then(writeVariableData)
```

Comment:
https://github.com/primer/css/pull/925 contains the async/await change to this script and this change will need to be included there